rgd/settings.py

```diff
@@ -104,6 +104,29 @@ class ProductionConfiguration(RgdMixin, ProductionBaseConfiguration):
     pass
 
 
+class AWSProductionConfiguration(RgdMixin, ProductionBaseConfiguration):
+    AWS_S3_ACCESS_KEY_ID = None
+    AWS_S3_SECRET_ACCESS_KEY = None
+    AWS_S3_REGION_NAME = None
+    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'http')
```

Comment:
This has the semantics that that "if `HTTP_X_FORWARDED_PROTO` has the value `http`, then the request is secure".

I don't know what the SSL-terminating proxy for this deployment is, but I would expect that this should be:
```suggestion
    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
```

Reply:
I'm not familiar with how this all works, but we are seeing infinite redirects when this is set to `https` on the AWS deployment. @chriswilley, can you provide any insight into the nginx setup for that deployment and what we are needing to set this to `http`? Perhaps, something with the nginx setup or load balancer needs to be changed?